tests/pagination/tests.py

```diff
@@ -304,6 +305,120 @@ def test_paginator_iteration(self):
             with self.subTest(page=page):
                 self.assertEqual(expected, list(next(page_iterator)))
 
+    def test_get_elided_page_range(self):
+        # Check that Paginator.validate_number() is called:
+        paginator = Paginator([1, 2, 3], 2)
+        with unittest.mock.patch.object(paginator, 'validate_number') as mock:
+            mock.assert_not_called()
+            list(paginator.get_elided_page_range(2))
+            mock.assert_called_with(2)
```

Comment:
@carltongibson Regarding my comment about testing exceptions, I've just remembered that I added this to check that `Paginator.validate_number()` is called which raises those exceptions for invalid page numbers. That is tested by `PaginationTests.test_invalid_page_number()` already.

Reply:
Okay, super, thanks for the clarification @pope1ni. I shall take another look tomorrow and hopefully that's job done! Good work, as ever. 😉